integration/__tests__/cluster-pages.tests.ts

```diff
@@ -364,9 +364,11 @@ utils.describeIf(minikubeReady(TEST_NAMESPACE))("Minikube based tests", () => {
     await kubeApiServerRow.click();
     await frame.waitForSelector(".Drawer", { state: "visible" });
 
-    const logButton = await frame.waitForSelector("ul.KubeObjectMenu li.MenuItem i.Icon span[data-icon-name='subject']");
+    const podLogsIcon = await frame.waitForSelector(".Drawer .drawer-title .Icon >> text=subject");
 
-    await logButton.click();
+    await podLogsIcon.click();
```

Comment:
nit this could just be

```suggestion
    await frame.click(".Drawer .drawer-title .Icon >> text=subject");
```

I don't know why I didn't do that initially.